src/otx/core/model/entity/base.py

```diff
@@ -42,6 +42,7 @@ def __init__(self, num_classes: int) -> None:
         self._label_info = LabelInfo.from_num_classes(num_classes)
         self.classification_layers: dict[str, dict[str, Any]] = {}
         self.model = self._create_model()
+        self.explain_hook = None
```

Comment:
If we move `register_explain_hook()` itself to the mixin class, does it look better to move this variable to the mixin class as well?

Reply:
Nevermind I'll make another PR to revise this overall